### ISSUE TITLE
Resources folder should be distributed with release archive

### ISSUE THREAD

**Issue by kudlav** [orzhaha/googlechat#1]
Resources folder should be distributed with a release archive. When missing the `resources` folder:

```
A JavaScript error occured in the main process

Error: Failed to load image from path
`C:/fubar/resources/icon200.png`
```

Downloading the `resources` folder fixed the problem.

![obrazek](https://github.com/orzhaha/googlechat/assets/11516259/13f9dd70-ea39-41c0-b37a-5471623f0673)

**Comment by kfrico** [comment]
Resources have been included in the [v0.0.4](https://github.com/orzhaha/googlechat/releases/tag/v0.0.4) release package. Thank you for the report.

**Comment by kudlav** [comment]
Thanks. But next time please clear the archive from the MacOS Finder hidden files. Each file has in the archive another one with a prefix `._`.
